web/source/dom/touchAliasElement.ts

```diff
@@ -398,6 +403,30 @@ namespace com.keyman.dom {
       this.showCaret();
     }
 
+    /**
+     * Facilitates higher-performance execution of caret position updates by disabling
+     * unnecessary DOM-manipulating (& layout-triggering) operations during the search.
+     * Some are still needed for the search to work, though.
+     * 
+     * Exists to encapsulate this behavior to ensure it's always, unfailingly re-enabled.
+     * @param functor A function that will search for the best caret position.
+     */
+    executeCaretSearchFunctor(functor: () => void) {
```

Comment:
Just `executeCaretSearch` is probably a good name? `Functor` is not really required.

I understand you need this indirection because the state variable is in touchAliasElement but the actual search is done in domEventHandlers, yes?

Reply:
I understand you need this indirection because the state variable is in touchAliasElement but the actual search is done in domEventHandlers, yes?

> Correct.